ext/fetch/22_body.js

```diff
@@ -222,9 +186,10 @@
      * @returns {InnerBody}
      */
     clone() {
+      this.#cloned = true;
```

Comment:
I don't understand why this is necessary. `this.stream.tee()` will return two fresh `ReadableStream` instances. After this operation neither `this.streamOrStatic` or `second.streamOrStatic` will be  FastStream sources anymore. Neither will have a `[_resourceBacking]` slot.

Reply:
It was necessary before using `readableStreamForRid` in my first implementation and forgot to test it again. I removed it since it is no longer needed as you mention.